prow/cmd/mkbuild-cluster/main.go

```diff
@@ -103,6 +107,26 @@ func main() {
 	}
 }
 
+// createDefaultClient creates and returns a default kubernetes clientset
+func createDefaultClient(ctx string) (*kubernetes.Clientset, error) {
+	config, err := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{
+			CurrentContext: ctx,
+		},
+	).ClientConfig()
+	if err != nil {
+		return nil, fmt.Errorf("get kube client config: %v", err)
```

Comment:
IMO `"get config: %v"` is fully sufficient here as there is not another config you need to disambiguate against.